OpenTabletDriver/Tablet/TabletReport.cs

```diff
@@ -21,7 +21,8 @@ public TabletReport(byte[] report)
             PenButtons = new bool[]
             {
                 report[1].IsBitSet(1),
-                report[1].IsBitSet(2)
+                report[1].IsBitSet(2),
+                report[1].IsBitSet(3)
```

Comment:
for consistent styling:
```suggestion
                report[1].IsBitSet(3),
```